depends/packages/qt.mk

```diff
@@ -8,7 +8,7 @@ $(package)_dependencies=openssl zlib
 $(package)_linux_dependencies=freetype fontconfig libxcb
 $(package)_build_subdir=qtbase
 $(package)_qt_libs=corelib network widgets gui plugins testlib
-$(package)_patches=fix_qt_pkgconfig.patch mac-qmake.conf fix_configure_mac.patch fix_no_printer.patch fix_rcc_determinism.patch xkb-default.patch no-xlib.patch
```

Comment:
14000 might need to include other changes. Not sure https://github.com/bitcoin/bitcoin/pull/14000/files

Reply:
It looks like all the changes are in. The patch file does look kinda wonky since the additions also have - next to them so it looks like things removed. strange but all changes are present.

Reply:
14000 was in effect backported here https://github.com/dashpay/dash/commit/3242f08e330b982cb95ab39c3c186a9fb2266169

Should drop 14000

Also @UdjinM6 reminded not to backport the backport, but backport the original PR :) Although I should have caught that in review of https://github.com/dashpay/dash/pull/3887

Reply:
oof, gotcha. So i should backport bitcoin#14005?

Reply:
No, that means we already backported 14005, which backports 14000 in bitcoin's codebase. Since we've merged 14005 there's no need to merge 14000